examples/auth/oauth2/static-route/manifests.yaml

```diff
@@ -57,7 +57,7 @@ spec:
             - name: HTTP_HEADERS
               value: 'Content-Type: "application/json"'
             - name: HTTP_BODY
-              value: '{ "kusk-gateway-manager": "v1.3.18" }'
+              value: '{ "kusk-gateway-manager": "v1.3.22" }'
```

Comment:
v1.3.20

Reply:
Can I leave it as `v1.3.22`?

The reason I am changing it is because `v1.3.20` is the current release.

Reply:
v1.3.21

Reply:
Resolved.

Now at `v1.3.21` in latest commit.

Reply:
```suggestion
              value: '{ "kusk-gateway-manager": "v1.3.21" }'
```